src/tools/genpolicy/rules.rego

```diff
@@ -1319,6 +1319,35 @@ ExecProcessRequest {
     print("ExecProcessRequest 3: true")
 }
 
+UpdateInterfaceRequest {
+    print("UpdateInterfaceRequest 1: input =", input)
```

Comment:
Here we have a single set of rules for UpdateInterfaceRequest, so "UpdateInterfaceRequest:" is enough - no need for "UpdateInterfaceRequest **1:**".